config/jacoco.gradle

```diff
@@ -32,15 +32,14 @@ task jacocoTestReport(type: JacocoReport) {
                             'com/nimbl3/ui/**/di/**',
                             'com/nimbl3/extension/**',
                             'com/nimbl3/**/extension/**']
-    // def filesExcluded = [ ]
-    // def fileFilter = fileGenerated + packagesExcluded + filesExcluded
+
     def fileFilter = fileGenerated + packagesExcluded
 
     classDirectories.from = fileTree(
-        dir: "$project.rootDir/app/build/intermediates/javac/stagingDebug/compileStagingDebugJavaWithJavac/classes",
+        dir: "$buildDir/intermediates/javac/stagingDebug/classes",
         excludes: fileFilter
     ) + fileTree(
-        dir: "$project.rootDir/data/build/intermediates/javac/stagingDebug/compileStagingDebugJavaWithJavac/classes",
+        dir: "$buildDir/intermediates/javac/stagingDebug/classes",
```

Comment:
@luongvo Sorry I just noticed the report when we use $buildDir/intermediates/javac/stagingDebug/classes we got this package  com.bumptech.glide also like this
![Screen Shot 2563-04-07 at 14 52 58](https://user-images.githubusercontent.com/28002315/78645744-311abf80-78e2-11ea-83a1-bfb23bc660d3.png)

Reply:
Can we exclude them out?

Reply:
Yes, we can, I will update @luongvo